src/app/Fake.IO.FileSystem/FileSystem.fs

```diff
@@ -1,304 +0,0 @@
-﻿/// Contains helpers which allow to interact with the file system.
-namespace Fake.IO.FileSystem
```

Comment:
Generally I don't feel we need to have a separate file for each module (this is not C#).

I'm not against splitting on some defined boundaries, for example we could split out the `Operators` or `File/FileInfo`, but as they are quite coupled I don't think that is too viable.

This module is probably one of the hardest to get into a nice state.